demo/components/presentational/LandRow.js

```diff
@@ -7,14 +7,22 @@ import {
 } from "react-native-responsive-dimensions";
 import Land from "./Land";
 
-export default function LandRow(props) {
-  return (
-    <TouchableHighlight onPress={props.onPress}>
-      <View style={styles.row}>
-        <Land land={props.land} />
-      </View>
-    </TouchableHighlight>
-  );
+// Note: Changing to PureComponent for performance boost
+// It is possible to still using function component with React.memo HoC
+// See more:
+// https://reactjs.org/docs/react-api.html#reactpurecomponent
```

Comment:
I think in React 16.7 there's a way to get the performance boost while keeping it written as a function. I'll try to dig up what I'm thinking of here. I'm pretty sure I'm thinking of something other than React.memo.

(I forget which React version React Native is using and which version of React Native Expo is using though)

Reply:
I've not found another way to use function components rather than React.memo.
We are using `react` 16.5.0
Expo 32.0.0 is using `react-native` v0.57.1 that uses `react` 16.5.0.

Reply:
Cool, thanks for the info! I'll resolve this for now and let you know if I figure out what I have in mind. Thanks!